airflow/providers/docker/operators/docker.py

```diff
@@ -222,7 +231,7 @@ def get_hook(self) -> DockerHook:
             tls=self.__get_tls_config(),
         )
 
-    def _run_image(self) -> Optional[str]:
+    def _run_image(self) -> Optional[Union[List[str], str]]:
```

Comment:
what is the convention?
```python
Optional[Union[List[str], str]]:
``` 
or
 
```python
Union[None,List[str], str]:
```


![Screenshot from 2022-02-08 18-36-03](https://user-images.githubusercontent.com/10202690/153043737-48e56a6c-d38b-402a-b642-b2c98912090c.png)

Reply:
Both work. But I personally prefer optional[Union] as it is more explicit IMHO.
 I am not sure we have agreed to a particular one :D